pkg/scheduler/core/generic_scheduler.go

```diff
@@ -45,6 +45,10 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/volumebinder"
 )
 
+const (
+	minFeasibleNodesToFind = 20
```

Comment:
Why is it 20, I think we need to have relevant comments to explain.

Reply:
20 is an arbitrary value. I added a comment to explain.